Source/WebCore/platform/image-decoders/png/PNGImageDecoder.cpp

```diff
@@ -197,7 +193,6 @@ PNGImageDecoder::PNGImageDecoder(AlphaOption alphaOption, GammaAndColorProfileOp
     : ScalableImageDecoder(alphaOption, gammaAndColorProfileOption)
     , m_doNothingOnFailure(false)
     , m_currentFrame(0)
-#if ENABLE(APNG)
     , m_png(nullptr)
```

Comment:
These should all be initializer lists in the header.  That prevents other constructors from forgetting to initialize them.

Reply:
That makes sense, but I'm going to leave that for somebody else to fix. Hope that's okay.